### PR TITLE
Disable HTML formatting by default

### LINKED ISSUE #1213: Disable HTML formatting by default
https://github.com/MyIntervals/emogrifier/blob/2923bdf5a78f2c8d65f961eb633bb4fd5c0cb46d/src/HtmlProcessor/AbstractHtmlProcessor.php#L243

Currently I'm solving it by using `::fromDomDocument` and passing custom document, wonder if there is a possibility of making it a built-in option/parameter?

### PR DESCRIPTION
**Pull request by osbre**
Closes #1213

### CODE REVIEW

**Review by oliverklee — CHANGES_REQUESTED** [review]
Now that I think of it, this seems to really solve a problem/bug for you. I'd like to avoid this problem creeping back in. So would you be willing to add a small regression test (that fails without the change and passes with the change)?

**Review by oliverklee — APPROVED** [review]
Beautiful. Thanks for contributing this, @osbre! :heart:

**Comment by oliverklee** [comment]
Oh, the PHP Sniffer is not happy. I'll fix this.

**Review by oliverklee — APPROVED** [review]

**Review by JakeQZ — COMMENTED** [review]
Sorry, I'm a bit late back to the party.

Good job both, particularly the thoughtful discussion about the rationale, motivation, whys and wherefores, etc.  I feel confident that the chosen solution is best.

However, I've spotted an issue with the `CHANGELOG` change which will need to be addressed.  I don't know if it's possible (or desirable) to make further changes in a PR which has already been approved and merged.

I've also added a couple of other comments which are somewhat neither here nor there, and nothing to worry about.

Thanks @osbre for contributing.

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,11 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 
 ### Fixed
 
+## 8.0.0
+
+### Changed
+- Disable HTML formatting by default (#1214)
+
```
[the remark this answers is not in the document]
Comment (osbre) [reply CHANGELOG.md:22]: @JakeQZ Thanks, made a PR: https://github.com/MyIntervals/emogrifier/pull/1215

File: tests/Unit/HtmlProcessor/AbstractHtmlProcessorTest.php [hunk 2]
```diff
@@ -108,6 +108,28 @@ public function renderPreservesOuterHtmlProvidedToFromHtml(): void
...
+        $rawHtml = '<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+        <title>Hello world</title>
+    </head>
+    <body>
+       <li><em>Hello</em> world</li>
+    </body>
+</html>';
```
[the remark this answers is not in the document]
Comment (osbre) [reply tests/Unit/HtmlProcessor/AbstractHtmlProcessorTest.php:125]: Thanks, made the PR: https://github.com/MyIntervals/emogrifier/pull/1216

**Pull request merged**